### PR TITLE
ENH: Start VTKPolyData traitlet

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: itkwidgets/trait_types.py [hunk 1]
```diff
@@ -242,3 +242,24 @@ def itkimage_from_json(js, manager=None):
     'from_json': itkimage_from_json,
     'to_json': itkimage_to_json
 }
+
+class VTKPolyData(traitlets.TraitType):
+    """A trait type holding an Python data structure compatible with vtk.js.
+
+    https://kitware.github.io/vtk-js/docs/structures_PolyData.html"""
```
Comment (jcfr) [inline itkwidgets/trait_types.py]: See https://kitware.github.io/vtk-js/docs/structures_PolyData.html

File: itkwidgets/trait_types.py [hunk 2]
```diff
@@ -242,3 +242,24 @@ def itkimage_from_json(js, manager=None):
     'from_json': itkimage_from_json,
     'to_json': itkimage_to_json
 }
+
+class VTKPolyData(traitlets.TraitType):
+    """A trait type holding an Python data structure compatible with vtk.js.
```
Comment (jcfr) [inline itkwidgets/trait_types.py]: `an Python data` -> `a Python data`

File: itkwidgets/trait_types.py (hunk 1 above)
Comment (thewtex) [reply itkwidgets/trait_types.py]: Added

File: itkwidgets/trait_types.py (hunk 2 above)
Comment (thewtex) [reply itkwidgets/trait_types.py]: Fixed

**Comment by thewtex** [comment]
@jcfr thanks for the review!

**Pull request merged**